Service/InstallationService.php

```diff
@@ -127,37 +151,95 @@ public function addActions(): void
             }
 
             $defaultConfig = $this->addActionConfiguration($actionHandler);
-
             $action = new Action($actionHandler);
+
             if ($schema['$id'] == 'https://vng.opencatalogi.nl/schemas/hp.availabilityCheck.schema.json') {
                 $action->setListens(['huwelijksplanner.calendar.listens']);
-            } elseif ($schema['$id'] == 'https://vng.opencatalogi.nl/schemas/hp.huwelijk.schema.json') {
-                $action->setListens(['huwelijksplanner.create.listens']);
-            } elseif ($schema['$id'] == 'https://vng.opencatalogi.nl/schemas/hp.assent.schema.json') {
-                $action->setListens(['huwelijksplanner.assent.listens']);
+                $action->setConditions([[1 => 1]]);
             } else {
                 $action->setListens(['huwelijksplanner.default.listens']);
             }
+
+            // set the configuration of the action
             $action->setConfiguration($defaultConfig);
+            $action->setAsync(false);
 
             $this->entityManager->persist($action);
 
-            (isset($this->io) ? $this->io->writeln(['Action created for '.$handler]) : '');
+            isset($this->io) && $this->io->writeln(['Action created for '.$handler]);
         }
     }
 
-    public function addDashboardCards()
+    private function createEndpoints($objectsThatShouldHaveEndpoints): array
     {
-        // Lets create some genneric dashboard cards
-        $objectsThatShouldHaveCards = [
-            'https://vng.opencatalogi.nl/schemas/hp.availabilityCheck.schema.json',
-            'https://commongateway.huwelijksplanner.nl/schemas/hp.huwelijk.schema.json',
-            'https://vng.opencatalogi.nl/schemas/hp.sdgProduct.schema.json',
-            'https://vng.opencatalogi.nl/schemas/hp.assent.schema.json',
+        $endpointRepository = $this->entityManager->getRepository('App:Endpoint');
+        $endpoints = [];
+        foreach ($objectsThatShouldHaveEndpoints as $objectThatShouldHaveEndpoint) {
+            $entity = $this->entityManager->getRepository('App:Entity')->findOneBy(['reference' => $objectThatShouldHaveEndpoint['reference']]);
+            if (!$endpointRepository->findOneBy(['name' => $entity->getName()])) {
+                $endpoint = new Endpoint($entity, $objectThatShouldHaveEndpoint['path'], $objectThatShouldHaveEndpoint['methods']);
```

Comment:
Take a good look at the new constructor for Endpoint here:
https://github.com/ConductionNL/commonground-gateway/blob/merge/into-cleanup/api/src/Entity/Endpoint.php 
And in the kiss-bundle:
https://github.com/Klantinteractie-Servicesysteem/KissBundle/blob/main/Service/InstallationService.php

Reply:
Also, if you see anyone use this old constructor while reviewing, please point them to these examples

Reply:
Without this change i will not approve this pr @bbrands02